assemblies/developer-lightspeed-guide/assembly_maas-oc-install-config.adoc

```diff
@@ -0,0 +1,32 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2025-04-08
+
+ifdef::context[:parent-context-of-maas-oc-install-config: {context}]
+
+:_mod-docs-content-type: ASSEMBLY
+
+ifndef::context[]
+[id="maas-oc-install-config"]
+endif::[]
+ifdef::context[]
+[id="maas-oc-install-config_{context}"]
+endif::[]
+= Installing and configuring {ocp-short} cluster
+:context: maas-oc-install-config
+
+As a member of the hybrid cloud infrastructure team, your initial set of tasks to deploy a large language model (LLM) through model-as-a-service involves creating {ocp-short} clusters with primary and secondary nodes and configuring an identity provider with role-based access control for users to log in to the clusters. 
```

Comment:
> As a member of the hybrid cloud infrastructure team

This seems strange, assume there was a mistake with including a section on "maas" for downstream docs.